nodeEditor/src/components/nodeList/nodeListComponent.tsx

```diff
@@ -33,7 +33,8 @@ export class NodeListComponent extends React.Component<INodeListComponentProps,
             Interpolation: ["LerpBlock", "SmoothStepBlock", "NLerpBlock"],
             Matrices: ["Matrix", "WorldMatrixBlock", "WorldViewMatrixBlock", "WorldViewProjectionMatrixBlock", "ViewMatrixBlock", "ViewProjectionMatrixBlock", "ProjectionMatrixBlock"],
             Mesh_Attributes: ["InstancesBlock", "PositionBlock", "UVBlock", "ColorBlock", "NormalBlock", "TangentBlock", "MatrixIndicesBlock", "MatrixWeightsBlock", "WorldPositionBlock", "WorldNormalBlock", "FrontFacingBlock"], 
-            Output_Blocks: ["VertexOutputBlock", "FragmentOutputBlock", "DiscardBlock"],
+            Noises: ["SimplexPerlin3DBlock"],
+			Output_Blocks: ["VertexOutputBlock", "FragmentOutputBlock", "DiscardBlock"],

```

Comment:
indent

Reply:
ahhh dumb dumb dumb, I forgot my notpad++ had tabs set to tabs not spaces on my laptop....